assets/ca/ca.querytranslator.js

```diff
@@ -81,15 +81,14 @@ var caUI = caUI || {};
 				case 'between':
 					return prefix + '[' + escapeValue(ruleSet.value[0]) + ' TO ' + escapeValue(ruleSet.value[1]) + ']';
 				case 'begins_with':
-					return prefix + '"' + escapeValue(ruleSet.value) + '"*';
+					return prefix + '' + escapeValue(ruleSet.value) + '*';
```

Comment:
Yes it will return what you say there. But given that starts with doesn't work at all with the quotes, this is a lesser bug to have.